opencraft/settings.py

```diff
@@ -282,7 +282,14 @@
 DEFAULT_CONFIGURATION_REPO_URL = env(
     'DEFAULT_CONFIGURATION_REPO_URL', default='https://github.com/edx/configuration.git'
 )
-DEFAULT_CONFIGURATION_VERSION = env('DEFAULT_CONFIGURATION_VERSION', default='master')
+
+DEFAULT_EDX_PLATFORM_REPO_URL = env(
+    'DEFAULT_EDX_PLATFORM_REPO_URL', default='https://github.com/edx/edx-platform.git'
```

Comment:
This is already information set by `DEFAULT_FORK`? If you want to use a URL for the repo instead, you would need to remove that setting.